<launch>

    <node name="pcloud_stich" pkg="visavis_vision" type="visavis_pcloud_stich" respawn="false"
        output="screen">
        <remap from="/in_pointcloud" to="/local_map" />

        <param name="voxel_size_stiching" value="0.1" />
        <param name="voxel_size_input_cloud" value="0.1" />
        <param name="max_cam_depth" value="3.0" />
        <param name="output_reference_frame" value="map" />
    </node>

    <node name="visavis_vision" pkg="visavis_vision" type="visavis_vision_node" respawn="false"
        output="screen">
        <param name="optical_frame" type="string" value="camera_color_optical_frame" />
        <param name="fixed_frame" type="string" value="map" />

        <param name="far_clip" type="double" value="4.0" />
        <param name="near_clip" type="double" value="0.01" />

        <!-- <param name="orig_cld_voxel_size" value="0.05" /> -->
        <param name="orig_cld_voxel_size" value="0.05" /> <!-- 0.1 -->

        <param name="object_cluster_distance" type="double" value="3" />
        <param name="min_object_cluster_size" type="int" value="1" />

        <remap from="in_cloud" to="/stiched_pointcloud" />

    </node>

</launch>